packages/cli/src/lib/bundler/config.ts

```diff
@@ -77,16 +79,65 @@ export function createConfig(
     }),
   );
 
+  const backendRelatedConfig = {
+    watch: true,
+    watchOptions: {
+      ignored: [/node_modules\/(?!\@backstage)/],
+    },
+    externals: [
+      nodeExternals({
```

Comment:
We don't want these to be external, we want to point directly to src and bundle these during the build

Reply:
Nah, node_modules you don't want to be processed, but all the `@backstage/*` things - sure. There is an adjustment for this in the second PR, but will move it over here

Reply:
Not sure tbh, I think it's pretty handy to be able to edit stuff in node_modules. Any idea what the performance impact would be like?

Reply:
I know for sure some node_modules failed to be bundled, can look into it a bit later

Reply:
Moved to #1238